spring-data-relational/src/main/java/org/springframework/data/relational/core/mapping/AggregatePath.java

```diff
@@ -240,7 +261,7 @@ record TableInfo(
 			 */
 			@Nullable SqlIdentifier tableAlias,
 
-			ColumnInfo reverseColumnInfo,
+			ColumnInfos reverseColumnInfos,
```

Comment:
Every time I see reverse column I'm having like no idea what this is. Also, the lack of documentation contributes to it. Expressing that this is the back reference column set to the owning entity would likely help.

Reply:
renamed to `backReferenceColumnInfos`. Also added proper JavaDoc comments and replaced the existing block comments on other fields to proper `@param` JavaDoc comments. 

I don't think we can have a proper `@since` annotation, so I made that part of the comment.